components/atoms/DateModified.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Contains build time stamp
+ */
+export default function DateModified(props) {
+  return (
+    <div property="dateModified">
+      Date Modified: {process.env.NEXT_PUBLIC_BUILD_DATE}
```

Comment:
Just noticed this after approving, insignificant but the first letter of Modified should not be capitalized in order to match what is on Canada.ca

There should also be 2em of padding on top and bottom, text should be Noto Sans, text color should be #333, font-size should be 16px, line-height should be 1.65em